### PR TITLE
build(language-service): Remove ls_rollup_bundle

### PR DESCRIPTION
**Pull request by kyliau**
`ls_rollup_bundle` is no longer needed since we could invoke `ng_rollup_bundle` directly.

Background: language service runs rollup to produce a single file to reduce
startup time in the editor. However, due to the need to load dynamic versions
of typescript at runtime (think the case where users can change typescript
version in their editor), we hack the "banner" to export a CommonJS default function,
so that we could dynamically load the typescript module provided at runtime 
by tsserver via AMD and use it throughout the implementation. 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/language-service/bundles/BUILD.bazel [hunk 1]
```diff
@@ -1,7 +1,9 @@
-load("//packages/language-service/bundles:index.bzl", "ls_rollup_bundle")
+load("//dev-infra/benchmark/ng_rollup_bundle:ng_rollup_bundle.bzl", "ng_rollup_bundle")
```
Comment (alfaproject) [inline packages/language-service/bundles/BUILD.bazel:1]: benchmark?

Comment (kyliau) [reply packages/language-service/bundles/BUILD.bazel:1]: This is directly copied over from the deleted file `packages/language-service/bundles/index.bzl`.

Comment (devversion) [reply packages/language-service/bundles/BUILD.bazel:1]: Yeah, we'll clean this up soon. The rollup bundle has been incorrectly placed under `benchmark`.

**Review by devversion — APPROVED** [review]
Nice cleanup! thanks!

**Pull request closed without merging**